projects/contracts/src/lib/types.ts

```diff
@@ -153,9 +153,17 @@ export interface AppState extends SettingsState {
 }
 
 export interface Config {
-  mediaFolder:   string;
+  mediaFolder: string;
   twitchChannel: string;
   twitchLog?: boolean;
+  twitch?: TwitchBotConfig
+}
+
+export interface TwitchBotConfig {
```

Comment:
I'll refactor this twitch config part in a separated PR and then we can merge this PR in